### DISCUSSION TITLE
Golang (Go) 🟢 Programming Language Thread

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#51388] [Programming Help]
[Golang](https://github.com/golang/go) or Go is an open source programming language you can use to build software. _$ denotes a paid resource._ 

**Around the web**

- [Go by Example](https://gobyexample.com/)
- [Tour of Go](https://go.dev/tour/welcome/1) 
- [Learn Go with Tests](https://quii.gitbook.io/learn-go-with-tests/) 
- [Gopher Exercises](https://gophercises.com/) 
- [Tech with Tim’s Youtube Golang Tutorial Series](https://www.youtube.com/playlist?list=PLzMcBGfZo4-mtY_SE3HuzQJzuj4VlUG0q) 

**Books**

- Learning Go - John Bodner
- Get Programming with Go - Nathan Youngman & Roger Peppé 

Share any links, tutorials, videos, and books that have helped you learn Go below - don’t forget to include if it is paid too!

**Comment by ibilalkayy** [comment]
**Channel**

[Golang Dojo](https://www.youtube.com/@GolangDojo)

**Video**

[Golang Tutorial for Beginners | Full Go Course](https://www.youtube.com/watch?v=yyUHQIec83I)

**Book**

Go in Action - William Kennedy, Brian Ketelsen, and Erik St. Martin

**Reply by queenofcorgis** [reply]
Thanks for sharing these resources!
